src/commands/run.js

```diff
@@ -28,4 +57,24 @@ RunCommand.description = 'Runs the act locally in the current directory.\n' +
     'the act input, as well as all other data in key-value stores, datasets or request queues to be stored in the "apify_local" directory, ' +
     'rather than on the Apify platform.';
 
+RunCommand.flags = {
+    'purge-stores': flagsHelper.boolean({
+        char: 'p',
+        description: 'It deletes all default local folder stores without "INPUT.*" before a run.',
```

Comment:
Better text: Shortcut that combines the --purge-queue, --purge-dataset and --purge-key-value-store options.